CMakeLists.txt

```
@@ -38,7 +38,7 @@ cmaize_option_list(
     INTEGRATION_TESTING OFF "Should we build the integration tests?"
 )
 
-set(DEPENDENCIES simde)
+set(DEPENDENCIES simde chemcache)
```

```suggestion
set(DEPENDENCIES simde)
```

Plugins shouldn't directly depend on other plugins like this. For testing just use a lambda module that returns the atomic info for the atoms you're testing.